### PR TITLE
Fix: product count exclude booking product

### PR DESCRIPTION
**Pull request by shohag121**
Product count is displaying count number including `product_type = booking` in this PR booking product are excluded.

fix: https://github.com/weDevsOfficial/dokan-pro/issues/1229

### CODE REVIEW

File: includes/functions.php [hunk 1]
```diff
@@ -186,7 +186,7 @@ function dokan_redirect_if_not_seller( $redirect = '' ) {
  *
  * @return array
  */
-function dokan_count_posts( $post_type, $user_id ) {
+function dokan_count_posts( $post_type, $user_id, $exclude_product_type = 'booking' ) {
```
Comment (nurul-umbhiya) [inline includes/functions.php:189]: make exclude_product_type to array and change code accordingly. in the future, we might need to exclude auction products also.

Comment (shohag121) [reply includes/functions.php]: Code changed as you instructed.

**Comment by FaraziF** [comment]
We can bring items counting on product listing page on frontend like backend https://prnt.sc/11ad5af

File: includes/functions.php [hunk 2]
```diff
@@ -183,23 +183,35 @@ function dokan_redirect_if_not_seller( $redirect = '' ) {
...
  */
-function dokan_count_posts( $post_type, $user_id ) {
+function dokan_count_posts( $post_type, $user_id, $exclude_product_types = array( 'booking' ) ) {
     global $wpdb;
 
-    $cache_group = 'dokan_seller_product_data_' . $user_id;
-    $cache_key   = 'dokan-count-' . $post_type . '-' . $user_id;
-    $counts      = wp_cache_get( $cache_key, $cache_group );
-
+    $exclude_product_types      = esc_sql( $exclude_product_types );
```
Comment (nurul-umbhiya) [inline includes/functions.php:193]: if you are using cache, you need to delete cache.

Comment (shohag121) [reply includes/functions.php:193]: Code changed as per your instruction.

**Review by nurul-umbhiya — APPROVED** [review]

**Pull request merged**